src/libsystemd/sd-journal/compress.c

```diff
@@ -533,7 +533,7 @@ int decompress_startswith(
                 return -EBADMSG;
 }
 
-int compress_stream_xz(int fdf, int fdt, uint64_t max_bytes) {
+int compress_stream_xz(int fdf, int fdt, uint64_t max_bytes, size_t *ret_uncompressed_size) {
```

Comment:
size_t looks like the wrong type. that might be 32bit, but we are operating on files here, hence should be fine with 64bit sizes: size_t → uint64_t please